src/frontends/paddle/docs/operation_mapping_flow.md

```diff
@@ -0,0 +1,13 @@
+# OpenVINO Paddle Frontend operator enabling Flow
```

Comment:
Most of diagrams contain outdated component names: nGraph, CNNNetwork. I think we definitely need to avoid it.

Reply:
Yes, it is why I asked to update all diagrams

Reply:
updated with mermaid.